File: jour_10/jou10.py
age = int(input("Entrez votre âge : "))
sexe = input("Entrez votre sexe (M/F) : ").upper()

# Traitement avec conditions combinées
if age >= 18 and age <= 25:
    print("Résultat : Vous bénéficiez du tarif 'Jeune Adulte'.")

elif age > 60 or (sexe == "F" and age > 55):
    print("Résultat : Vous bénéficiez du tarif 'Sénior'.")
elif age < 18:
    print("Résultat : Vous bénéficiez du tarif 'Enfant'.")
else:
    print("Résultat : Tarif standard appliqué.")

File: jour_10/jou10.c
#include <stdio.h>
#include <ctype.h>

int main() {
    int age;
    char sexe;

    printf("Entrez votre âge : ");
    scanf("%d", &age);
    printf("Entrez votre sexe (M/F) : ");
    scanf(" %c", &sexe);
    sexe = toupper(sexe);

    // Conditions combinées 
    if (age >= 18 && age <= 25) {
        printf("Résultat : Tarif 'Jeune Adulte'.\n");
    } 
    else if (age > 60 || (sexe == 'F' && age > 55)) {
        printf("Résultat : Tarif 'Sénior'.\n");
    } 
    else if (age < 18) {
        printf("Résultat : Tarif 'Enfant'.\n");
    } 
    else {
        printf("Résultat : Tarif standard.\n");
    }

    return 0;
}